docs/using-the-compiler.rst

```diff
@@ -127,12 +127,12 @@ Input Description
         //   evm.ir - New assembly format before desugaring
         //   evm.assembly - New assembly format after desugaring
         //   evm.legacyAssemblyJSON - Old-style assembly format in JSON
-        //   evm.opcodes - Opcodes list
-        //   evm.methodIdentifiers - The list of function hashes
-        //   evm.gasEstimates - Function gas estimates
         //   evm.bytecode - Bytecode
+        //   evm.bytecode.opcodes - Opcodes list
```

Comment:
Please add `evm.bytecode.object`.

Further, how does the output selector look like if I only want to have the object but not the list of opcodes? Note that the list of opcodes is useless most of the time since it does not work once you have data like the metadata hash.